python/cuml/test/utils.py

```diff
@@ -123,3 +124,12 @@ def normalize_clusters(a0, b0, n_clusters):
 def clusters_equal(a0, b0, n_clusters):
     a, b = normalize_clusters(a0, b0, n_clusters)
     return array_equal(a, b)
+
+
+def get_handle(use_handle):
```

Comment:
What is the `use_handle == False` cases good for? If I don't want a handle I simply don't call `get_handle`, right?

Reply:
the unit-tests for dbscan add 2 more cases of users trying to pass an already created `cuml.Handle` vs an implicitly created one inside the `Base` class constructor. This helper method just avoids those repeated if-else statements in each of the unit-tests. What say?

Reply:
Make sense. Thanks for clarifying.